api/routes/productRouter.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const { productController } = require("../controllers");
+
+const router = express.Router();
+
+router.get("/", productController.productList);
```

Comment:
```suggestion
router.get("", productController.productList);
```
REST API에서는 uniform interface 원칙에 따라 엔드포인트 끝에 (`/`)가 들어가는 것을 지양하고 있습니다!